js/src/annotations/osd-svg-rectangle.js

```diff
@@ -1,13 +1,11 @@
 (function($) {
   $.Rectangle = function(options) {
-    jQuery.extend(this, {
+    jQuery.extend(this, options ,{
```

Comment:
In the other shape files, `options` is the last parameter for `extend`, but in this file you moved it